

Invoking dbt with ['seed', '--select', 'seed', '--full-refresh']
[0m16:50:20  Running with dbt=1.7.18
[0m16:50:20  Registered adapter: firebolt=1.6.4
[0m16:50:20  Found 12 models, 1 seed, 0 sources, 0 exposures, 0 metrics, 420 macros, 0 groups, 0 semantic models
[0m16:50:20  
[0m16:50:21  Concurrency: 1 threads (target='default')
[0m16:50:21  
[0m16:50:21  1 of 1 START seed file seed .................................................... [RUN]
[0m16:50:22  1 of 1 OK loaded seed file public.seed ......................................... [[32mCREATE 6[0m in 1.13s]
[0m16:50:22  
[0m16:50:22  Finished running 1 seed in 0 hours 0 minutes and 2.10 seconds (2.10s).
[0m16:50:22  
[0m16:50:22  [32mCompleted successfully[0m
[0m16:50:22  
[0m16:50:22  Done. PASS=1 WARN=0 ERROR=0 SKIP=0 TOTAL=1


Invoking dbt with ['run', '--select', 'not_found_unique_key', '--full-refresh']
[0m16:50:22  Running with dbt=1.7.18
[0m16:50:22  Registered adapter: firebolt=1.6.4
[0m16:50:22  Found 12 models, 1 seed, 0 sources, 0 exposures, 0 metrics, 420 macros, 0 groups, 0 semantic models
[0m16:50:22  
[0m16:50:23  Concurrency: 1 threads (target='default')
[0m16:50:23  
[0m16:50:23  1 of 1 START sql incremental model not_found_unique_key ........................ [RUN]
[0m16:50:24  1 of 1 OK created sql incremental model not_found_unique_key ................... [[32mSUCCESS[0m in 1.04s]
[0m16:50:24  
[0m16:50:24  Finished running 1 incremental model in 0 hours 0 minutes and 2.06 seconds (2.06s).
[0m16:50:24  
[0m16:50:24  [32mCompleted successfully[0m
[0m16:50:24  
[0m16:50:24  Done. PASS=1 WARN=0 ERROR=0 SKIP=0 TOTAL=1


Invoking dbt with ['run', '--select', 'not_found_unique_key']
[0m16:50:24  Running with dbt=1.7.18
[0m16:50:24  Registered adapter: firebolt=1.6.4
[0m16:50:24  Found 12 models, 1 seed, 0 sources, 0 exposures, 0 metrics, 420 macros, 0 groups, 0 semantic models
[0m16:50:24  
[0m16:50:25  Concurrency: 1 threads (target='default')
[0m16:50:25  
[0m16:50:25  1 of 1 START sql incremental model not_found_unique_key ........................ [RUN]
[0m16:50:27  1 of 1 ERROR creating sql incremental model not_found_unique_key ............... [[31mERROR[0m in 1.32s]
[0m16:50:27  
[0m16:50:27  Finished running 1 incremental model in 0 hours 0 minutes and 2.33 seconds (2.33s).
[0m16:50:27  
[0m16:50:27  [31mCompleted with 1 error and 0 warnings:[0m
[0m16:50:27  
[0m16:50:27    Runtime Error in model not_found_unique_key (models/not_found_unique_key.sql)
  Error executing query:
  {
    "errors": [
      {
        "description": "Line 3, Column 17: Column 'thisisnotacolumn' does not exist."
      }
    ],
    "query": {
      "query_id": "94da48d9-c412-4484-a6c3-f1abfc9ad5f5",
      "request_id": "5d09e160-fe46-4e87-8c87-bdb5843afed6"
    },
    "statistics": {
      "elapsed": 0.0
    }
  }
  
[0m16:50:27  
[0m16:50:27  Done. PASS=0 WARN=0 ERROR=1 SKIP=0 TOTAL=1
